openpype/hosts/blender/plugins/create/create_blendScene.py

```diff
@@ -0,0 +1,51 @@
+"""Create a Blender scene asset."""
+
+import bpy
+
+from openpype.pipeline import get_current_task_name
+from openpype.hosts.blender.api import plugin, lib, ops
+from openpype.hosts.blender.api.pipeline import AVALON_INSTANCES
+
+
+class CreateBlendScene(plugin.Creator):
+    """Generic group of assets"""
+
+    name = "blendScene"
+    label = "Blender Scene"
+    family = "blendScene"
+    icon = "cubes"
+
+    def process(self):
+        """ Run the creator on Blender main thread"""
+        mti = ops.MainThreadItem(self._process)
+        ops.execute_in_main_thread(mti)
+
+    def _process(self):
```

Comment:
When does this one get executed? And should we have the `import bpy` here instead of at the top?